extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -93,7 +96,18 @@ require([
 
 	function performRequest($elem) {
 		var requestUrl = getBaseUrl() + $elem.attr('data-requestUrl'),
-			requestData = JSON.parse($elem.attr('data-requestData'));
+			requestData = JSON.parse($elem.attr('data-requestData')),
+			columnsDetailsClass;
+
+		// Inject proper class for 2 columns display
+		if (requestData.columns === 2) {
+			if ($elem.closest('.main-page-tag-rcs').length) {
+				// When the tag is inside the right rail
```

Comment:
This comment should clarify you mean the main page right rail/column, not the normal right rail on article pages.